BD-4465 WhatsApp videos supported in Media Library (#9382)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/whatsapp/whatsapp_campaign/create.md

Headers | 
Text | Supports variable parameters
Images (JPEG and PNG) | Must be 8-bit, RGB or RGBA, and up to five MB for any type 

> Videos | Can be hosted on a URL or in the Media Library. Must be less than sixteen MB and have a video type of 3GPP or MP4. <br><br>Video messages are currently in early access. Contact your Braze account manager if you're interested in participating in the early access.

Reviewer: ```suggestion
Videos | Supports hosting videos through URL or the Media Library. Must be less than sixteen MB and have a video type of 3GPP or MP4. <br><br>Video messages are currently in early access. Contact your Braze account manager if you're interested in participating in the early access.
```

Note: what is "Media Library" and is it a proper noun? If its a Braze product, could we add a hyperlink to the text?

Reviewer 2: Media library is one of those terms that has historically been capitalized, and now that we have established some guidelines for consistency, is no longer to be capitalized. There are still some areas of the product that need to be updated to follow the correct capitalization, but you can check the [Braze Glossary](https://docs.google.com/document/d/1Sllt9uoteI_bRpbbNYsHt3TdQ9pCkgY-NgQBg6Gmbjo/edit?tab=t.0) (maintained by marketing) for the main source of truth.

Reviewer: @bre-fitzgerald thanks! so it is a braze feature then? i'm assuming this is the related info?

https://www.braze.com/docs/user_guide/engagement_tools/templates_and_media/media_library

Reviewer 2: correct!

> Videos | Can be hosted on a URL or in the Media Library. Must be less than sixteen MB and have a video type of 3GPP or MP4. <br><br>Video messages are currently in early access. Contact your Braze account manager if you're interested in participating in the early access.

Reviewer 2: Also, we should use numeral for measurements (16 MB not sixteen MB)
```suggestion
Videos | Supports hosting videos through URL or the [media library]({{site.baseurl}}/user_guide/engagement_tools/templates_and_media/media_library). Must be less than 16 MB and have a video type of 3GPP or MP4. <br><br>Video messages are currently in early access. Contact your Braze account manager if you're interested in participating in the early access.
```

Reviewer: 100% just changed it in my commit 💪🏽
